src/shared/components/ObjectsTable/ObjectsTable.js

```diff
@@ -192,6 +210,28 @@ const ObjectsTable = ({
     setCurrentFilter(id);
   };
 
+  const arrowOnClick = (clickedRow) => {
+    const rowIndex = sortedRows.findIndex((row) => row.key === clickedRow.key);
+    const expanded = !clickedRow.expanded;
+    const newRows = [
+      ...sortedRows.slice(0, rowIndex),
+      {
+        ...clickedRow,
+        expanded,
+      },
+      ...sortedRows.slice(rowIndex + 1, sortedRows.length),
+    ];
+
```

Comment:
```
    const newRows = [
      {
        ...clickedRow,
        expanded,
      },
    ];
```
wouldn't be enough?

Reply:
or action `UPDATE_OR_ADD_OBJECT` with object (without wrapping in an array) as a payload
```
      {
        ...clickedRow,
        expanded,
      },
```

Reply:
Ah you're right, the redux action already filters the array so no need to pass everything back.
Fixed in 84f1ca3